### ISSUE TITLE
File paths with spaces not resolving

### ISSUE THREAD

**Issue by JJSax** [lwz7512/love2d-made-easy#5]
In my exact case, I have my open file path inside a folder named "Genetic Algorithm Tests".  When launching Love2d, the path it says in the love2d crash screen.

```
Error

[love "boot.lua"]:323: Cannot load game at path 'c:/Users/jauge/OneDrive/Documents/Programs/Love2d/Genetic'.
Make sure a folder exists at the specified path.


Traceback

[love "callbacks.lua"]:228: in function 'handler'
[C]: in function 'error'
[C]: in function 'xpcall'
[C]: in function 'xpcall'
```

You'll notice the folder name ends at the first space, which is incorrect.  

I'm also getting an error when using alt+L to launch saying "command 'loveme.launch' not found", though this might be another extension.

**Comment by mintysylvie** [comment]
Yup! I had this issue too. Project was inside a folder named "Coding Projects". Failed to find main.lua. Changed to "CodingProjects" and it works finally! Even if this never gets fixed, you saved me the hassle of finding a different extension

**Comment by ShinuToki** [comment]
I had the same error but I managed to solve it modifying a simple line of the source code of the extension. 
In the line 73 of the file `Love2d.js` you need to replace this:
```
child.exec(`${love2dExePath} ` + projectRoot).on("close", function (code, signal) {
```
for this:
```
child.exec(`${love2dExePath} ` + `"${projectRoot}"`).on("close", function (code, signal) {
```

**Comment by lwz7512** [comment]
> Yup! I had this issue too. Project was inside a folder named "Coding Projects". Failed to find main.lua. Changed to "CodingProjects" and it works finally! Even if this never gets fixed, you saved me the hassle of finding a different extension

thanks for your support!

**Comment by lwz7512** [comment]
> I had the same error but I managed to solve it modifying a simple line of the source code of the extension. In the line 73 of the file `Love2d.js` you need to replace this:
> 
> ```
> child.exec(`${love2dExePath} ` + projectRoot).on("close", function (code, signal) {
> ```
> 
> for this:
> 
> ```
> child.exec(`${love2dExePath} ` + `"${projectRoot}"`).on("close", function (code, signal) {
> ```

Awesome! i never had a chance to fix it, thanks for your contribution, i'll make patch for this fix!

**Comment by lwz7512** [comment]
issue fixed with new release: v1.10.3 

please check it out!

thanks for feedback!
